### PR TITLE
added accessibility helpers

### PR DESCRIPTION
**Pull request by ateliercw**
addresses #17

### CODE REVIEW

File: Pod/Classes/AccessibilityHelpers/AccessibilityHelpers.swift [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    /**
+     Announce a message via VoiceOver with optional completion callback (UIAccessibilityAnnouncementNotification).
+
+     - parameter text:       The text to be spoken.
+     - parameter completion: A block to be invoked when the announcement has completed.
+     */
+    public func announce(_ text: String, completion: AccessibilityAnnounceCompletion? = nil) {
+        announceCompletion = completion
```
Comment (ahtierney) [inline Pod/Classes/AccessibilityHelpers/AccessibilityHelpers.swift]: #? are there any race conditions here where we could be calling this several times in a row and overwriting our completion? (could very well be left to programmer error)? Maybe we can check to see if we're in progress or similar?

Comment (ateliercw) [inline Pod/Classes/AccessibilityHelpers/AccessibilityHelpers.swift]: @nbonatsakis / @ZevEisenberg any input on if we need to worry about this?

Comment (ZevEisenberg) [inline Pod/Classes/AccessibilityHelpers/AccessibilityHelpers.swift]: As a compromise, we could call a pending completion before reassigning it. But we could als make a FIFO queue.

**Comment by ateliercw** [comment]
set up a FIFO announcement queue, @ZevEisenberg thoughts?

File: Pod/Classes/AccessibilityHelpers/AccessibilityHelpers.swift [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+//  AccessibilityHelpers.swift
+//  Pods
+//
+//  Created by Michael Skiba on 11/17/16.
+//
+//
+
+import UIKit
+
+public typealias AccessibilityAnnounceCompletion = (_ anncouncedString: String?, _ success: Bool?) -> ()
```
Comment (ZevEisenberg) [inline Pod/Classes/AccessibilityHelpers/AccessibilityHelpers.swift]: #nth I just replaced this `Bool?` with a `Bool` on my project, and used `[get the bool from the dictionary] ?? false` where it's called. Optional Booleans are annoying, and they make the implementation of the completion into this mess:

```swift
if let success = success, success {
    // what even
}
```

**Comment by ZevEisenberg** [comment]
@ateliercw what if you announce a long thing and then a short thing, and the short thing finishes first? It will call the completions out of order. Might just want to keep around a single previous completion, and flush it before calling another announcement. Let's talk IRL if that's not clear. Typing words are failing me this morning.

File: Pod/Classes/AccessibilityHelpers/AccessibilityHelpers.swift [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    /**
+     Announce a message via VoiceOver with optional completion callback (UIAccessibilityAnnouncementNotification).
+
+     - parameter text:       The text to be spoken.
+     - parameter completion: A block to be invoked when the announcement has completed.
+     */
+    public func announce(_ text: String, completion: AccessibilityAnnounceCompletion? = nil) {
+        concurrentAnnouncementQueue.sync {
```
Comment (ahtierney) [inline Pod/Classes/AccessibilityHelpers/AccessibilityHelpers.swift]: 👍

**Pull request merged**